### DISCUSSION TITLE
A new JDA docs website ??

### DISCUSSION THREAD

**Discussion by vedjain773** [discord-jda/JDA#1727] [Questions and Help]
A good library often requires a good website that lists all its methods, classes and functions. As far as my experience is concerned, JDA is one of the best Java libraries out there. The problem with most if not all Java libraries is that they lack a proper documentation of their respective libraries. It would be really nice if the JDA docs website improves, it would help me and hopefully hundreds of others who use this library. I know the thing that I'm asking for is a bit large for a project but it would make the user's experience a lot better while using this library.

A good example for a Docs Website would be the one, discord.js has:-
[https://discord.js.org/#/](https://discord.js.org/#/)

**Comment by MinnDevelopment** [comment]
We offer very detailed javadocs here: [Documentation](https://ci.dv8tion.net/job/JDA/javadoc/)
We offer guides for various topics in our wiki: [Wiki](https://github.com/DV8FromTheWorld/JDA/wiki)
We also offer example classes: [Examples](https://github.com/DV8FromTheWorld/JDA/tree/development/src/examples/java)

There is not much more we can offer for a library.
